lib/src/functions/meta.dart

```diff
@@ -77,6 +79,18 @@ final local = UnmodifiableListView([
             ? argument
             : SassString(argument.toString(), quotes: false)),
         ListSeparator.comma);
+  }),
+  _function("accepts-content", r"$mixin", (arguments) {
+    var mixin = arguments[0].assertMixin("mixin");
+    bool acceptsContent = switch (mixin.callable) {
+      AsyncBuiltInCallable(hasContent: var hasContent) ||
+      BuiltInCallable(hasContent: var hasContent) ||
+      UserDefinedCallable(declaration: MixinRule(hasContent: var hasContent)) =>
+        hasContent,
+      AsyncCallable() =>
+        throw UnsupportedError("Unknown callable type $mixin."),
```

Comment:
Unsure what should happen here. AFAICT this class has been replaced by `AsyncBuiltInCallable` and is never itself actually used anywhere?

Reply:
Throwing an `UnsupportedError` is correct. Eventually hopefully https://github.com/dart-lang/language/issues/3113 will be addressed and we'll be able to explicitly declare to the language that the set of callables described above are exhaustive.